python/spacewalk/spacewalk-backend.changes

```diff
@@ -1,3 +1,5 @@
+- Fix reposync content-type key bug
```

Comment:
Just small nitpick here: let's try to be a bit more specific with the changelog message:

```suggestion
- Fix reposync error about missing "content-type" key when syncing certain channels
```

Reply:
Oh, sorry, meant to put more detail in and forgot, thanks!